docs-requirements.txt

```diff
@@ -44,61 +34,50 @@ incremental==22.10.0
     # via towncrier
 jinja2==3.1.3
     # via
-    #   -r docs-requirements.in
     #   sphinx
     #   towncrier
 markupsafe==2.1.5
     # via jinja2
 outcome==1.3.0.post0
-    # via -r docs-requirements.in
```

Comment:
Are these expected? or missing compatibility from `uv`?

Reply:
`uv` doesn't appear to explicitly list the fact a dependency is requested from the `.in` file, which kind of makes sense. If nothing is listed as a what required that module, why waste text? It also makes it more immediately clear when something is a requirement of another package.

Additionally, `uv` hasn't explicitly said that they are trying to have 100% compatibility for these compiled files, and I personally am for this particular change differing from `pip-compile`.

Reply:
> If nothing is listed as a what required that module, why waste text?

It'd make senses when multiple `.in` passed for single output.
Also, it'd be increasing readability for recognizing top-level requirements.

Reply:
@charliermarsh Do you think this is compatibility problem and need to open issue for it? or this is by design/not planned?

Reply:
nvm. found the issue: https://github.com/astral-sh/uv/issues/1343

Reply:
This is now fixed as of https://github.com/astral-sh/uv/pull/3269